app/views/desks/index.html.erb

```diff
@@ -1,7 +1,24 @@
-<h3>Desk list</h3>
+<%= stylesheet_link_tag 'index' %>
```

Comment:
You don't need to include the `index.scss`. If you import it in `aplications.scss` it will be able to use in this file.

Reply:
?

Reply:
This [link on SO](https://stackoverflow.com/questions/10250468/for-a-rails-view-html-erb-where-can-i-place-the-style-sheet) does a good job of explaining how to include assets.